src/_main_/settings.py

```diff
@@ -24,7 +24,7 @@
 # ********  LOAD CONFIG DATA ***********#
 IS_PROD = False
 IS_CANARY = False
-IS_LOCAL = False
+IS_LOCAL = True
```

Comment:
try to avoid committing this or DEBUG=True

Reply:
Noted